mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -244,6 +246,7 @@ object Tool extends LazyLogging {
 
   private def runCheck(executor: PassChainExecutor, check: CheckCmd): Int = {
     setCoreOptions(executor, check)
+    print(executor.options.get[List[String]]("checker", "temporal"))
```

Comment:
This looks like some left-over debug output?

Reply:
Yes, thanks!